translations/strings.yaml

```diff
@@ -595,6 +595,72 @@ tipsAndTricks:
   quickTipsDescription: Let us help you get the most out of Mozilla VPN
   quickTipsGuideViewTitle: Quick tip
 
+subscriptionManagement:
+  menuTitle:
+    value: Profile
+    comment: Menu title for the a view with details on the users account.
+  summaryHeadline:
+    value: Subscription summary
+    comment: Headline for a list with information on the subscription plan of the user.
+  paymentHeadline:
+    value: Payment Information
+    comment: Headline for a list with information on the payment information the user provided for their subscription.
+  planLabel:
+    value: Subscription plan
+    comment: Label for the selected subscription plan the user currently is subscribed to.
+  planValueMonthly:
+    value: "%1 monthly"
+    comment: The %1 represents a localized currency string
+  planValueHalfYearly:
+    value: "%1 half-yearly"
+    comment: The %1 represents a localized currency string
+  planValueYearly:
+    value: "%1 yearly"
+    comment: The %1 represents a localized currency string
+  statusLabel:
+    value: Status
+    comment: Label for the current status of the subscription.
+  statusInactive:
+    value: Inactive
+    comment: Text for a label that indicates an inactive subscription.
+  statusActive:
+    value: Active
+    comment: Text for a label that indicates an active subscription.
+  nextLabel:
+    value: Next billed
+    comment: Label for the upcoming date on which the user will be billed for their subscription.
+  activatedLabel:
+    value: Activated
```

Comment:
Would it make more sense to have something like "Start date", "Activation date", or "Active since"?

Reply:
I like these more descriptive suggestions. What do you think @peterbenvenuto and @chrisrooneys?